cdap-app-fabric/src/test/java/io/cdap/cdap/internal/app/services/http/handlers/NamespaceHttpHandlerTest.java

```diff
@@ -383,6 +387,16 @@ public void testNamespaceClient() throws Exception {
     } catch (NotFoundException expected) {
       // expected
     }
+
+    //test metrics are expected
+    MetricsManager metricsManager = new MetricsManager(getInjector().getInstance(MetricStore.class));
```

Comment:
Can you do a metrics check before and after the delete on line 383? This will ensure we test both creation and deletion

Reply:
Done.